pkg/template/api/types.go

```diff
@@ -23,23 +30,22 @@ type Template struct {
 type Parameter struct {
 	// Required: Parameter name must be set and it can be referenced in Template
 	// Items using ${PARAMETER_NAME}
-	Name string `json:"name"`
+	Name string
```

Comment:
It's something we need to adapt to, upstream no longer uses JSON tags on internal types because you aren't supposed to serialize them (why we have external).